src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -131,11 +131,16 @@ class CornerstoneViewport extends Component {
       viewport: cornerstone.getDefaultViewport(null, undefined)
     };
 
-    const { loadHandlerManager } = cornerstoneTools;
-    loadHandlerManager.setStartLoadHandler(this.startLoadingHandler);
-    loadHandlerManager.setEndLoadHandler(this.doneLoadingHandler);
-
     this.debouncedResize = debounce(() => {
+      const { loadHandlerManager } = cornerstoneTools;
```

Comment:
Why did you move these inside the resize handler? Put them back where they were, it makes no sense to re-set these every single time the element is resized.

Reply:
yes, sir. Done